packages/server/src/instances/events.ts

```diff
@@ -0,0 +1,21 @@
+import { Emitter } from '../base/events'
+import { uuid } from '../base/types'
+
+// Note: event type ids should be random numbers between -9007199254740991 and 9007199254740991
```

Comment:
And why is that?

Reply:
That's a copy pasted comment from earlier in development. I wanted to make a sort a meta event watcher but I ended up not doing that. I'll remove all these random enum values in another PR and just let the autoincremented number enums instead